src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -311,31 +311,30 @@ module Network_manager = struct
   let run_cmd_exn t prog args = run_cmd_exn t.testnet_dir prog args
 
   let create ~logger (network_config : Network_config.t) =
+    let%bind all_namespaces =
+      Cmd_util.run_cmd_exn "/" "kubectl"
+        ["get"; "namespaces"; "--output='json'"]
+    in
+    let contains_substring str sub =
+      let re = Str.regexp_string sub in
+      Str.string_match re str 0
+    in
+    let%bind _ =
+      if
+        contains_substring all_namespaces network_config.terraform.testnet_name
+      then
+        Cmd_util.run_cmd_exn "/" "kubectl"
```

Comment:
Style nit: to avoid the `_` unbound pattern and the need to return `""` on the else branch, you can write it like this:

```ocaml
let%bind () =
  if ... then
    Cmd_util.run_cmd_exn "/" "kubectl" ["delete"; "namespace"] >>| Fn.const ()
  else
    Deferred.unit (* or `return ()` *)
```

Reply:
cool, figured there was a better way to write that